RobotCode/src/main/java/frc/robot/commands/ShooterOverrideCommand.java

```diff
@@ -0,0 +1,47 @@
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj2.command.*;
+import frc.robot.constants.IndexerConstants;
+import frc.robot.constants.ShooterConstants;
+import frc.robot.subsystems.IndexerSubsystem;
+import frc.robot.subsystems.ShooterSubsystem;
+
+public class ShooterOverrideCommand extends Command
+{
+	// Initializes the subsystem objects
```

Comment:
These lines do not initialize the objects. The objects are initialized in the constructor.

Reply:
Still not resolved